content/docs/guides/scim/azuread.md

```diff
@@ -0,0 +1,117 @@
+---
+title: Configuring SCIM in Azure Active Directory
+meta_desc: This page describes how to support SCIM 2.0 functionality between Pulumi and Azure AD.
+menu:
+    userguides:
+        parent: scim
+        weight: 1
+---
+
+This document outlines the steps required to configure automatic provisioning/deprovisioning of your users in Pulumi using SCIM 2.0.
+
+Please note that some advanced SCIM features aren't supported yet. For more information, see [Known Limitations]({{< relref "#known-limitations" >}}).
+
+## Prerequisites
+
+* Your organization must already be configured to use [SAML SSO]({{< relref "/docs/guides/saml/aad" >}}) with Pulumi.
+* You must be an admin of your Pulumi organization.
+* (Optional, but highly recommended) You should have more than one admin for your Pulumi organization.
+
+## Enabling Automatic Provisioning
+
+* Navigate to the Azure Active Directory where you have configured Single Sign On using SAML with Pulumi.
+* Click on **Enterprise Applications** and select the app in which you configured Single Sign On with Pulumi earlier.
+* Then, click on the **Provisioning** feature, and change the value of **Provisioning Mode** to **Automatic**.
+
+### Admin Credentials
+
+Under the **Admin Credentials** section of the **Provisioning** feature, fill out the form as follows:
+
+* **Tenant URL**: `https://api.pulumi.com/scim/v2/{orgName}`, where `{orgName}` must be replaced with your organization’s login name (not display name). If you do not know this, navigate to your SAML settings and look at the SSO URL. It will have your organization’s login name in the URL.
+* **Secret Token**: You will use a token from the [Pulumi Console](https://app.pulumi.com) as the authorization bearer token. To generate a token, navigate to your org in the Pulumi Console, click on the **Settings** tab, and then click **SAML SSO**. Scroll down to the **SCIM** section and generate a new token if you have never generated one for your org, or regenerate it if you have already done so in the past.
+
+    {{% notes "info" %}}
+Once you generate the token, please save it securely. Neither the Pulumi Console nor Pulumi support can retrieve a token once it's been initially generated. If you lose the SCIM token and need it again, you'll have to generate a new token, which invalidates any previous tokens for your Pulumi organization.
+    {{% /notes %}}
+
+* Click the **Test Connection** button. You should get a success notification once the connection is successful.
```

Comment:
Since this is a solo step, it doesn't need to be included in the bulleted list as an item to fill out.

```suggestion
Click **Test Connection**. You should get a success notification once the connection is successful.
```